src/core/__tests__/pick.test.js

```diff
@@ -0,0 +1,20 @@
+import 'babel-polyfill'
+import { expect } from 'chai'
+import { pick } from '../'
+import Immutable from 'immutable'
+
+const object = { 'a': 1, 'b': 2, 'c': 3 }
```

Comment:
These are good basic tests but may want to make use of the new testing helpers for testing against multiple types. Check out the tests in [`at.test.js`](https://github.com/brianneisler/mudash/blob/master/src/core/__tests__/at.test.js)